packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -155,7 +263,7 @@ class DateUtil {
 
   /**
    * Converts an ISO string to the given format.
-   * @param {string} iSODate - The ISO string to convert.
+   * @param {T[]} iSODate - The ISO string to convert.
```

Comment:
What does `T` stand for?

Reply:
This is a code format correction from the ide I've been using that I didn't notice. :/ I'll swap it back!